bin/kano-capture-logs

```diff
@@ -38,13 +45,16 @@ def get_log_size():
 
 def get_boot_space():
     lines = subprocess.check_output(
-        "df -k /boot/ --output=avail".split(" ")).split("\n")
+        "df -k /boot/ --output=avail".split(" ")
+    ).split("\n")
     return int(lines[1])
 
 
-get_logs()
+if __name__ == '__main__':
+
+    get_logs()
 
-if (get_boot_space()-get_log_size()) > SPACE_MARGIN:
-    os.system("cp {} /boot".format(LOG_DEST))
-    os.system("syncfs </boot/config.txt")
-    play_sound(SOUND_FILE)
+    if (get_boot_space() - get_log_size()) > SPACE_MARGIN:
+        os.system("cp {} /boot".format(LOG_DEST))
+        os.system("syncfs </boot/config.txt")
```

Comment:
Is this `syncfs` meant to flush `config.txt` or the logfile?

Reply:
I don't know which is intended, but it applies to the whole filesystem so it will do both.